CapitalLettersParametrizerTests/src/test/java/LettersFormatterTest.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+
+class LettersFormatterTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"RYSIEK", "KRZYSIEK", "OLA", " "})
```

Comment:
A jakiś scenariusz na to, że tekst zawiera małe litery? ;) Aktualnie nie testujesz tego wystarczająco, bo równie dobrze metoda może wyglądać tak:
```
public static String changeToUpperCase(String text){

        if (text == null) {
            text = "";
        }

    return text //czyli nic nie robimy
    }
```
a twój test nadal powie 'jest ok, wszystko dziala'.